docs/source/trust-context-didcomm.rst

```diff
@@ -0,0 +1,117 @@
+Trusted Contexts via DIDComm
+============================
+
+Summary
+-------
+
+This paper highlights the conceptual gaps between the existing Aries credentials exchange model
+where the User is the Holder of the credentials being requested, and the TrustBloc model where the
+User is not the Holder, yet retains authority over the issuance of these credentials. It introduces
+new concepts to bridge these gaps that build upon existing Aries protocols and ideas.
+
+Motivation
+----------
+
+The fundamental difference between the :ref:`trusted_contexts` model of user data exchange and the
```

Comment:
The `ref` does not render right in GitHub, but Sphinx correctly renders it as a link to the "Trusted Contexts" section in `msg-routing-storage.rst`